### ISSUE TITLE
Fails to configure-make in separate build folder (../configure && make)

### ISSUE THREAD

**Issue by slime73** [love2d/love#617]
**[Original report](https://bitbucket.org/rude/love/issue/612) by Anonymous.**

----------------------------------------

Steps to reproduce:

1. $ hg clone https://bitbucket.org/rude/love
2. cd love
3. platform/unix/automagic
4. mkdir build && cd build
5. ../configure
6. make -j1

What happens:
Build fails with compile error:
 ./../src/libraries/Box2D/Common/b2Draw.cpp:19:33: fatal error: Box2D/Common/b2Draw.h: No such file or directory

In other words, the common method of creating a build folder to preserve source directory, and running the configure script from this separate folder breaks the build.

Expected results:
That it works the same as ./configure && make (which works fine).

Additional information not applicable, as this should be a general unix issue.
